staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -42,10 +42,15 @@ import (
 type AdmissionOptions struct {
 	// RecommendedPluginOrder holds an ordered list of plugin names we recommend to use by default
 	RecommendedPluginOrder []string
-	// DefaultOffPlugins a list of plugin names that should be disabled by default
-	DefaultOffPlugins []string
-	PluginNames       []string
-	ConfigFile        string
+	// DefaultEnabledPluginSet is a set of plugin names that is enabled by default
+	DefaultEnabledPluginSet sets.String
```

Comment:
DefaultEnabledPlugins. No need for the Set suffix.

Reply:
ok